backend/src/main/java/gov/cdc/usds/simplereport/api/pxp/PatientExperienceController.java

```diff
@@ -126,4 +133,18 @@ public void patientLinkSubmit(
     ps.updateMyTestResultDeliveryPreference(data.getTestResultDelivery());
     pls.expireMyPatientLink();
   }
+
+  // PostAuthorize will need to be updated once there is a concept of a general patient registration
+  // user that can be used for audit logging
+  @PreAuthorize("permitAll()")
+  @PostAuthorize("permitAll()")
+  @GetMapping("/register/entity-name")
+  public String getEntityName(
+      @RequestParam String patientRegistrationLink, HttpServletRequest request) {
+    PatientRegistrationLink link = prls.getPatientRegistrationLink(patientRegistrationLink);
+    if (Objects.nonNull(link.getFacility())) {
```

Comment:
Super minor nitpick here: [Per the javadoc](https://docs.oracle.com/javase/8/docs/api/java/util/Objects.html#nonNull-java.lang.Object-), `Objects.nonNull` is intended to be used is a Predicate, e.g. `filter(Objects::nonNull)`.

It would be more conventional to write `if(link.getFacility() != null)` outside of that context